src/web/components/App.tsx

```diff
@@ -299,6 +302,38 @@ export const App = ({ CAPI, NAV }: Props) => {
                     <CalloutBlockComponent callout={callout} pillar={pillar} />
                 </Hydrate>
             ))}
+            {CAPI.qandaAtoms.map((qandaAtom) => (
+                <Hydrate root="qanda-atom" index={qandaAtom.qandaIndex}>
+                    <QandaAtom
+                        id={qandaAtom.id}
+                        title={qandaAtom.title}
+                        html={qandaAtom.html}
+                        image={qandaAtom.img}
+                        credit={qandaAtom.credit}
+                        likeHandler={() => {
+                            recordComponentEvent(
+                                'LIKE',
+                                qandaAtom.id,
+                                'QANDA_ATOM',
+                            );
+                        }}
+                        dislikeHandler={() => {
+                            recordComponentEvent(
+                                'DISLIKE',
+                                qandaAtom.id,
+                                'QANDA_ATOM',
+                            );
+                        }}
+                        expandHandler={() => {
+                            recordComponentEvent(
+                                'EXPAND',
+                                qandaAtom.id,
+                                'QANDA_ATOM',
+                            );
+                        }}
+                    />
+                </Hydrate>
```

Comment:
As per https://github.com/guardian/atoms-rendering/pull/28/files#r460810610 you can probably pass an `ophanHandler` as single prop, and then deal with the function names inside the atom